solidity/test/bridge/Bridge.MovingFunds.test.ts

```diff
@@ -2249,6 +2327,20 @@ describe("Bridge - Moving funds", () => {
                                   ).to.be.equal(await lastBlockTime())
                                 })
 
+                                it("should decrease the sweeping wallet's pending requests count", async () => {
+                                  // The `setPendingMovedFundsSweepRequest` call
+                                  // made as part of `runMovedFundsSweepScenario`
+                                  // set this counter to 1. Eventually, it
```

Comment:
What about replacing "Eventually" with "After submitting the proof of sweeping the moved funds"?